CMakeLists.txt

```diff
@@ -109,8 +109,17 @@ if (BUILD_SDF)
   ign_find_package(ignition-utils1 VERSION REQUIRED)
   set(IGN_UTILS_VER ${ignition-utils1_VERSION_MAJOR})
 
+  ########################################
+  # Find ignition common
+  ign_find_package(ignition-common4 COMPONENTS graphics REQUIRED_BY usd)
+  set(IGN_COMMON_VER ${ignition-common4_VERSION_MAJOR})
```

Comment:
We need to have a dependency on `ign-common` in order to parse meshes to/from SDF and USD. I realize that this is a new dependency that we probably don't want to have, but since it's only needed by the `usd` component, is it okay to have this new dependency?

Reply:
yes, it's ok